application/src/utils/PostMessage.js

```diff
@@ -158,7 +158,9 @@ export function sendPostMessage(event, data, scheme = 'mx') {
       if(window === window.parent){
         console.log('Post message: ' + postUrl);
         console.log(message)
-        window.parent.postMessage(message, postUrl)
+        if(window.JSChannel){
```

Comment:
I see a condition is added, thanks!
although, I think it should be more effective to move this condition upper a level and not restricted to line 158